### DISCUSSION TITLE
Faker import error

### DISCUSSION THREAD

**Discussion by seenanair** [faker-js/faker#1190] [Q&A]
I was using faker package. But  now it is deprecated, I installed @faker-js/faker 7.3.0 using yarn in my React/Typescript project.  But I am getting the following error "Attempted import error: '@faker-js/faker' does not contain a default export (imported as 'faker'). 
 faker-js documentation says "Since version v6+ there is native TypeScript support.". Any pointers will be greatly appreciated.

**Comment by xDivisionByZerox** [comment]
Please have a look at [our usage documentation](https://fakerjs.dev/guide/#node-js).
I'm very sure you imported faker via the default value:
```
import faker from '@faker-js/faker';
```

This is deprecated. You are required to use that named export:
```
import { faker } from '@faker-js/faker';
```

If that was not the error please let me know.

**Reply by seenanair** [reply]
Thank you for your reply.  Unfortunately . I already tried import { faker } from '@faker-js/faker'; But it was giving the same error.

import { faker } from "@faker-js/faker/locale/en"; seems to work. Is this a right way of usage?

**Reply by xDivisionByZerox** [reply]
> Thank you for your reply. Unfortunately . I already tried import { faker } from '@faker-js/faker'; But it was giving the same error.

That's odd. Are you able to link to your source code or set up a minimal reproduction of your workspace for further inspections?

> import { faker } from "@faker-js/faker/locale/en"; seems to work. Is this a right way of usage?

Well, it's the (currently) intended way when importing a faker instance from a specific locale. So, if you are fine with the English version of faker you should be good to go.

Alternatively you could create the default instance as we do in our [index.ts](https://github.com/faker-js/faker/blob/main/src/index.ts).
```
import { Faker } from '@faker-js/faker';
import allLocales from '@faker-js/faker/locales';

const faker: Faker = new Faker({
  locales: allLocales,
});
```

**Reply by seenanair** [reply]
This is a very good suggestion rather than importing from a specific locale and it seems working. Thank you very much. Much appreciated. 👍
